manifests/kube_addons.pp

```diff
@@ -25,28 +25,39 @@
   }
 
   if $cni_rbac_binding {
-    $shellsafe_binding = shell_escape($cni_rbac_binding)
+    $binding_command = ['kubectl', 'apply', '-f', shell_escape($cni_rbac_binding)]
```

Comment:
Ah, you don't have to shell_escape if it's being passed through like this.
Sorry if I gave you a bad example earlier.